### DISCUSSION TITLE
[question]  How to search by line feed code by using exact-match?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [community/community#136822] [Discussions] [answered]
### Select Topic Area

Question

### Body

There has below code:
```
imports: {
    importFromNode: "import_from_statement",
```

I want to query it by using exact-match [`""`](https://docs.github.com/en/search-github/github-code-search/understanding-github-code-search-syntax#query-for-an-exact-match),  

but how about the line feed of it ? 

I only can search on GitHub by: `imports: {    importFromNode: "import_from_statement",`

can not search using:

```
imports: {
    importFromNode: "import_from_statement",
```

**Comment by agustrodmar** [comment, marked as answer]
Hello there! 
It would try to adapt my approach because Github search engine doesn't support direct line break searches.

I would remove the line breaks and searching for the code as if it were a sigle line. Try:

`imports: { importFromNode: "import_from_statement",`

Github is pretty limited for this stuff, I guess.
